packages/app/src/cli/services/app-context.test.ts

```diff
@@ -232,7 +232,7 @@ describe('linkedAppContext', () => {
       const meta = metadata.getAllPublicMetadata()
       expect(meta).toEqual(
         expect.objectContaining({
-          partner_id: tryParseInt(mockRemoteApp.organizationId),
+          business_platform_id: tryParseInt(mockOrganization.id),
           api_key: mockRemoteApp.apiKey,
           cmd_app_reset_used: false,
```

Comment:
We probably should test that `partner_id` is no longer present in this case? since `objectContaining` will still succeed even if present.

(or add a new test for the partner_id case)

Reply:
Good idea, I added that below.